extensions/amp-apester-media/0.1/amp-apester-media.css

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* apester-iframe */
+
+.apester-iframe {
+  transition: 0.4s opacity;
+  opacity: 0;
+}
+
+.apester-iframe-ready {
+  opacity: 1 !important;
+}
+
+.apester-loader {
+  height: 100px;
+  width: 100px;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  webkit-transform: translate(-50%, -50%);
```

Comment:
I commented on this earlier: no need for vendor prefixes - our CSS compiler adds them for you.